src/source/PeerConnection/JitterBuffer.c

```diff
@@ -238,6 +181,87 @@ STATUS jitterBufferPop(PJitterBuffer pJitterBuffer, BOOL bufferClosed)
     return retStatus;
 }
 
+
+STATUS freeJitterBuffer(PJitterBuffer* ppJitterBuffer)
+{
+    ENTERS();
+
+    STATUS retStatus = STATUS_SUCCESS;
+    PJitterBuffer pJitterBuffer = NULL;
+
+    CHK(ppJitterBuffer != NULL, STATUS_NULL_ARG);
+    // freeJitterBuffer is idempotent
+    CHK(*ppJitterBuffer != NULL, retStatus);
+
+    pJitterBuffer = *ppJitterBuffer;
+
+    jitterBufferInternalParse(pJitterBuffer, TRUE);
+    jitterBufferDropBufferData(pJitterBuffer, 0, MAX_SEQUENCE_NUM, 0);
+    hashTableFree(pJitterBuffer->pPkgBufferHashTable);
+
+    MEMFREE(*ppJitterBuffer);
```

Comment:
I see this was there before but for this to be idempotent we need to do:

`SAFE_MEMFREE(*ppJitterBuffer);`

Which will set `*ppJitterBuffer` to NULL

Reply:
Ah! I wasn't aware of that API change. I'll fix that.